sound/soc/intel/boards/sof_sdw.c

```diff
@@ -631,7 +631,6 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.direction = {true, true},
 		.dai_name = "max98373-aif1",
 		.init = sof_sdw_mx8373_init,
-		.codec_card_late_probe = sof_sdw_mx8373_late_probe,
```

Comment:
Can you use separated commits for the changes?
1. remove `late_probe` and test if `if (codec_info_list[i].codec_card_late_probe)`.
2. The max98373 stuff

Reply:
Ack, will split it into 2 as suggested, thanks!